apps/api/infra/api.ts

```diff
@@ -161,6 +161,7 @@ export const serviceSetup = (services: {
       Education,
       NationalRegistry,
     )
+    .files('islyklar.p12')
```

Comment:
They usually need an env variable with the path to the file similar to [this](https://github.com/island-is/helm/blob/main/charts/identity-server/values.yaml#L30). Make this a hash where the key is the env variable name and the file is the value or the other way around 🙃

Reply:
fair enough

Reply:
done